packages/backend/src/app.ts

```diff
@@ -232,6 +234,8 @@ export interface AppServices {
   incomingPaymentService: Promise<IncomingPaymentService>
   remoteIncomingPaymentService: Promise<RemoteIncomingPaymentService>
   receiverService: Promise<ReceiverService>
+  grantService: Promise<GrantService>
+  authServerService: Promise<AuthServerService>
```

Comment:
weren't included before, adding here so we get proper types

Reply:
easy to miss this